### ISSUE TITLE
HightCharts- How to hide hidden heading tags from Chart 

### ISSUE THREAD

**Issue by MdAmeerMind** [highcharts/highcharts#13291]
#### Expected behavior
Narrator/NVDA should not read “Chart, Long Description, Structure, Chart graphic” headings where no headings are visible on the page.

#### Actual behavior
Due to compliance of accessibility, heading structure should be maintain and `<h>` tags should be visible.
While using Windows Narrator Turn on Scan mode (Caps Lock + Space) and press H. For NVDA press H. 
Narrator/NVDA is reading “Chart, Long Description, Structure, Chart graphic” headings where no headings are visible on the page.

#### Live demo with steps to reproduce
[https://www.highcharts.com/blog/tutorials/182-7-most-common-chart-mistakes/](https://www.highcharts.com/blog/tutorials/182-7-most-common-chart-mistakes/)

Now start NVDA , press H and observe whats narrator is pronouncing. 

<img width="878" alt="HighChart" src="https://user-images.githubusercontent.com/48435553/78892034-3d3f8200-7a86-11ea-9ccb-79fcc46bf43b.PNG">

So there is no use of highlighted content on image as its `<h>` tag and hidden.
How to hide or configure settings to hide heading tags from chart? 

```
import React, { Component } from 'react';
import ReactHighcharts from 'react-highcharts';
import Accessibility from 'highcharts/modules/accessibility';
class MyCompo extends Component {
	constructor(props) {
		super(props);
	}	

	componentDidMount() {
		Accessibility(ReactHighcharts.Highcharts);
	}

	render() {
		return (
			<React.Fragment>
                              <ReactHighcharts config={this.getPieChartConfiguration()}>
                             </ReactHighcharts>
			</React.Fragment>
		);
	}

	onLegendClick(data) {
		
	}

	getPieChartConfiguration() {
		var pieColors = [];		
		return {
			credits: {
				enabled: false
			},
			chart: {
				type: 'pie',
				backgroundColor: 'rgb(244,244,244)',
				spacingTop: 0,
				spacingRight: 0,
				spacingBottom: 0,
				spacingLeft: 0,
				margin: [0, 0, 0, 0]
			},
			accessibility: {
				description: 'Overview Charts'
			},
			title: {
				text: ''
			},
			tooltip: {
				pointFormat: '<b>{point.y:.0f}</b>'
			},
			plotOptions: {
				pie: {
					allowPointSelect: true,
					cursor: 'pointer',
					depth: 35,
					colors: pieColors,
					dataLabels: {
						enabled: false,
						format: '{point.name},{point.y:.0f}'
					},
					point: {
						events: {
							click: (event) => {
								//this.onLegendClick(event.target.point.data);
							}
						}
					}
				}
			},
			series: [
				{
					type: 'pie',
					name: 'Count',
					data: //data
				}
			],
			lang: {
				accessibility: {
					chartContainerLabel: 'Test'
						,
					svgContainerTitle:
						'Test'
				}
			}
		};
	}
}
```

#### Product version
react-highcharts: 16.0.2

**Comment by pawelfus** [comment]
Thanks for reporting @MdAmeerMind 

@oysteinmoseng - could you take a look? Thanks!

**Comment by oysteinmoseng** [comment]
@MdAmeerMind In the latest versions of Highcharts, this behavior has changed, and there is now only one heading element per chart as default, which corresponds to the chart title. This behavior is configurable using the [accessibility.screenReaderSection.beforeChartFormat](https://api.highcharts.com/highcharts/accessibility.screenReaderSection.beforeChartFormat) option.

**Comment by MdAmeerMind** [comment]
Hi @oysteinmoseng , thank you for your response. 
I have updated `react-highcharts` version to `16.1.0`. but still having same issue.
My observation with react version `16.1.0` is, its 7 month old and still referring old version fo `highcharts` may be (6.X.X) internally for which issue doesn't have fix.
I think just update `react-highcharts` version wont fix this issue.
I am changing my code for and moving from `react-highcharts` to `highcharts 8.0.4`,  it have fixes for the issue.

**Comment by oysteinmoseng** [comment]
@MdAmeerMind Great to hear that your issue is fixed in the latest version. Note that `react-highcharts` is not an official package maintained by us. For the official Highcharts React package, see https://www.npmjs.com/package/highcharts-react-official.
